### DISCUSSION TITLE
Source code of theme keycloak.v3

### DISCUSSION THREAD

**Discussion by parvuselephantus** [keycloak/keycloak#28321] [Q&A] [answered]
Hi, I have a heavly modified account theme, which had parent=base. I failed to make it visible in any way until modifying - applying parent=kecyloak.v3, also had to introduce index.ftl. My starting point was keycloak v20.0.5. The issues I'm having:
- I lost support for multiple variables - it seems this is due to change of server support -> client support. I see a post is being sent the get all the values (like email, name, ...). That leads me to my next problem:
- If I wanted to rebuild my account theme based on keycloak.v3 I found org.keycloak.keycloak-account-ui-24.0.1.jar but it has built version of react project. I can't find anywhere the source version - can somebody please point me the source code for keycloak.v3 so I could fork it and reimplement my theme?

**Comment by justin-tay** [comment, marked as answer]
The source code should be located at https://github.com/keycloak/keycloak/tree/main/js/apps/account-ui/src

**Comment by parvuselephantus** [comment]
Thank you, can somebody please confirm my path is correct: I can't use anything else but keycloak.v3 as parent? It is yet another brick of complication for me to have yet another step of compilation/building of theme.

**Reply by justin-tay** [reply]
You might want to refer to the following issue

* https://github.com/keycloak/keycloak/issues/27771

**Reply by parvuselephantus** [reply]
Thank you for your help. It seems I will have lot's of pain now with keycloak but thanks to you at least I think I will know where to start.
